gantry/firmware/can_task.cpp

```diff
@@ -38,6 +40,13 @@ extern FDCAN_HandleTypeDef fdcan1;
 static auto can_bus_1 = HalCanBus(&fdcan1);
 static auto message_writer_1 = MessageWriter(can_bus_1);
 
+static std::map<GantryAxisType, NodeId> AxisToNodeIdMap{
```

Comment:
this is probably a bad idea since it allocates on the heap, sadly enough. it can probably be this instead though:

```
static constexpr NodeId node_from_axis(GantryAxisType which) {
   switch (which) {
      case GantryAxisType::gantry_x: return NodeId::gantry_x;
      case GantryAxisType::gantry_y: return NodeId::gantry_y;
   }
}
```

the compiler will yell at you if you leave off a case because it's an enum, and it probably gets fully optimized out, and then you can call it inline below.